File: examples/university-demo-OLD/src/main/java/io/axoniq/demo/university/faculty/write/subscribestudentmulti/SubscribeStudent.java
package io.axoniq.demo.university.faculty.write.subscribestudentmulti;

import io.axoniq.demo.university.faculty.write.CourseId;
import io.axoniq.demo.university.faculty.write.StudentId;

public record SubscribeStudent(StudentId studentId, CourseId courseId) {

}

File: examples/university-demo-kotlin/src/main/kotlin/faculty/write/subscribe_student/SubscribeStudent.kt
package io.axoniq.demo.university.faculty.write.subscribe_student

import io.axoniq.demo.university.faculty.ids.CourseId
import io.axoniq.demo.university.shared.ids.StudentId

data class SubscribeStudent(
  val studentId: StudentId,
  val courseId: CourseId,
)
